src/library_exceptions.js

```diff
@@ -265,52 +309,53 @@ var LibraryExceptions = {
   // functionality boils down to picking a suitable 'catch' block.
   // We'll do that here, instead, to keep things simpler.
 
-  __cxa_find_matching_catch__deps: ['__exception_last', '__exception_infos', '__resumeException'],
+  __cxa_find_matching_catch__deps: ['__exception_last', 'ExceptionInfo', '__resumeException'],
```

Comment:
Found that this declarations actually had no effect because addCxaCatch() function below did not handle it.